File: samples/jafu-servlet-minimal/src/main/java/com/sample/Application.java
package com.sample;

import org.springframework.fu.jafu.JafuApplication;

import static org.springframework.fu.jafu.Jafu.webApplication;
import static org.springframework.fu.jafu.webmvc.WebMvcServerDsl.webMvc;

public class Application {

	public static JafuApplication app = webApplication(a -> a.beans(b -> b
			.bean(SampleHandler.class)
			.bean(SampleService.class))
			.enable(webMvc(s -> s
					.port(s.profiles().contains("test") ? 8181 : 8080)
					.router(router -> {
						SampleHandler handler = s.ref(SampleHandler.class);
						router
								.GET("/", handler::hello)
								.GET("/api", handler::json);
					}).converters(c -> c
							.string()
							.jackson()))));

	public static void main (String[] args) {
		app.run(args);
	}
}

File: samples/kofu-servlet-validation/src/main/kotlin/com/sample/Application.kt
package com.sample

import org.springframework.fu.kofu.webApplication
import org.springframework.fu.kofu.webmvc.webMvc

val app = webApplication {
    beans {
        bean<UserHandler>()
        bean(::routes)
    }
    webMvc {
        port = if (profiles.contains("test")) 8181 else 8080
		converters {
			jackson()
		}
    }
}

fun main() {
    app.run()
}